devel/bazel/Portfile

```diff
@@ -49,8 +49,10 @@ distname                  ${distname}-dist
 dist_subdir               ${name}
 
 # Checksum Phase
-checksums                 rmd160  5269d2fce917d3324dcb75ca06fe12089a4e24a1 \
-                          sha256  2418c619bdd44257a170b85b9d2ecb75def29e751b725e27186468ada2e009ea
+checksums                 rmd160  ed9c36b49d60aaead2d16bc28ece7c026745d632 \
+                          sha256  dfd0761e0b7e36c1d74c928ad986500c905be5ebcfbc29914d574af1db7218cf
+depends_build-append \
+    bin:bash:bash
```

Comment:
Why did you add this?

Reply:
I didn't actually add this but I can see why @emcrisostomo did. The compile script needs to be run with the `bash` executable.

Reply:
macOS >= 10.5 should have bash with the system, so I think `bin:bash:bash` would be pointless.

Reply:
@l2dy I'm afraid I lost a notification here. As @RootFunction pointed out, I added it because `bash` is required by the build infrastructure. I've been wondering whether or not to add that build dependency, and finally erred on the side of adding it. My first choice was actually adding a port dependency to `bash`, but I finally added `bin:bash:bash`. When writing that section I consulted the MacPorts documentation and found this:

> The most commonly-used type of dependencies in Portfiles are port dependencies, because dependencies should be provided by MacPorts ported software whenever possible, and usually only one port can provide the needed libraries and files.
>
> But when satisfying a dependency with vendor-supplied software is preferred for special reasons, or when it is possible for more than one port to satisfy a dependency, then file dependencies may be used. An example of the former is with ubiquitous utilities like awk, grep, make or sed, where OS X's versions are often sufficient; an example of the latter is with “-devel” ports—these ports provide a different version of the same files (though only one can be activated at a time).

So, if I understand that section correctly, even if the system provides a `bash` binary, it doesn't seem a sufficient reason _not_ to include a `bin:bash:bash` dependency. Maybe it's overkill, but this way the port is clear in stating its needs and is also more robust.

Reply:
OK, I understand.